man/systemd.network.xml

```diff
@@ -2461,9 +2450,40 @@ IPv6Token=prefixstable:2002:da8:1::</programlisting></para>
       </varlistentry>
       <varlistentry>
         <term><varname>BindToInterface=</varname></term>
-        <listitem>Takes a boolean value. When <literal>yes</literal>, DHCP server socket will be bound
-        to its network interface and all socket communication will be restricted to this interface.
-        Defaults to <literal>yes</literal>.
+        <listitem>
+          <para>Takes a boolean value. When <literal>yes</literal>, DHCP server socket will be bound
+          to its network interface and all socket communication will be restricted to this interface.
+          Defaults to <literal>no</literal>, if <varname>RelayTarget=</varname> is set(see below),
+          otherwise defaults to <literal>yes</literal>.</para>
```

Comment:
still missing space before `(`.

i#d also reverse the sentence, mention the common case first. i.e.

Defaults to <literal>yes</literal>, except if <varname>RelayTarget=</varname> is used in which case it defaults to <literal>no</literal>